roles/rp/tasks/main.yml

```diff
@@ -37,7 +37,7 @@
   sudo: true
   docker_container:
     name: rp
-    image: jrgm/123done
+    image: jrgm/123done{{ ':' + rp_docker_tag }}
```

Comment:
we should file an issue on 123done, can we pull the changes required to make 123done built with circle ci?